### PR TITLE
Improve setup

### PR DESCRIPTION
**Pull request by rgaiacs**
Addresses #111 and #110.

### CODE REVIEW

**Comment by acabunoc** [comment]
Hey @rgaiacs -- thanks for looking into this!

So, we have a couple pr's (#87) open for adding setup information into 01-select.md. This information was moved out of 00-setup.md and into discussion.md right before out release. (by @mckays630 71f5f37acc5cb6ab929fb7d4a8f11654aab3da6d) as requested by @gvwilson [I think] (https://github.com/swcarpentry/sql-novice-survey/pull/87#issuecomment-106469643)

Since more than one person has been confused by this, I think it might be a good idea to re-instate 00-setup.md (or at least add more setup info to the first lesson)

If we don't hear any more input on this pr or #87, I'll make a maintainer decision next week.

File: instructors.md [hunk 1]
```diff
@@ -105,117 +105,28 @@ to make it easier for scientists to find what they want to.
...
 In order to execute the following lessons interactively,
-please install SQLite as mentioned in the setup instructions for your workshop,
-then create a directory `swc_sql`:
+please install SQLite as mentioned in the setup instructions for your workshop.
+Then
 
 ~~~ {.bash}
-$ mkdir swc_sql
-$ cd swc_sql
+$ git clone http://github.com/swcarpentry/sql-novice-survey.git
```
Comment (iglpdc) [inline instructors.md]: @rgaiacs I'd prefer to have a zip file for getting the file from a zip file: the same as the rest of the lessons. The current instructions require `wget` which is not installed by default neither in OSX nor in Windows.

This PR's require `git`, which is better, but can be a problem, e.g. if someone wants to teach this lesson before the Git lesson (or in a Mercurial workshop).

**Comment by iglpdc** [comment]
Related to #94

File: instructors.md (hunk 1 above)
Comment (rgaiacs) [inline instructors.md]: @iglpdc I understand your point and I am OK with any maintainer decision. =)

Comment (iglpdc) [inline instructors.md]: Actually, I see now that **I** made a PR long ago (#83) to use the zip file similarly to the other lessons :)

Comment (acabunoc) [inline instructors.md]: I'd also prefer zip! Sorry for the delay in my maintainer duties! Will catch up soon

**Comment by gvwilson** [comment]
Status?

**Comment by acabunoc** [comment]
merging this, but we may want to update instructors.md with zip file instructions

**Pull request merged**